apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -37,6 +42,11 @@
                 "ping_interval" : {
                     "description" : "Interval between pings (in ms)",
                     "type" : "integer"
+                },
+                "external_port" : {
```

Comment:
Could you please integrate [full sample config](https://github.com/aeternity/epoch/blob/214908f39c986d1d70bfbf64a971a435ffa6387c/apps/aeutils/test/data/epoch_full.yaml) with this param?